### DISCUSSION TITLE
Selecting a date in year 0-99AD returns 19XX year.

### DISCUSSION THREAD

**Discussion by ccorcos** [gpbl/react-day-picker#1108] [Support] [answered]
**Describe the bug**
It's impossible to specify a date in the first century.

**To Reproduce**
Just try it: [CodeSandbox](https://codesandbox.io/s/XDAE3x0W8)

**Comment by KyorCode** [comment, marked as answer]
This is actually due to javascript date behavior. Try it in your console `new Date(95, 1, 1)`  and the year will be 1995, as you stated. 

I think the easiest way to overcome is, is to use the parseDate method to correctly parse these dates.
https://react-day-picker.js.org/api/DayPickerInput#parseDate

That in combination for example with date-fns/setYear:

```
const setYear = require('date-fns/setYear')

const d = setYear(new Date(95,1,1), 95);
```

**Comment by ccorcos** [comment]
Is that something I need to do on my end or is that internal to the library?

**Comment by KyorCode** [comment]
On your end. But I do agree it should be handle by the date library the component uses.
